templates/fine-tune-llm_v2/cookbooks/continue_from_checkpoint/README.md

```diff
@@ -0,0 +1,80 @@
+# Continue fine-tuning from a previous checkpointing
+
+**⏱️ Time to complete**: 40 minutes
+
+This guide assumes that you have familiarized yourself with the main fine-tuning guide of this template.
+In this cookbook tutorial, we showcase how a checkpoint that was created earlier can be used as initialization for another round of fine-tuning.
+This allows us to sequentially combine fine-tuning on multiple datasets in order to get performance boost on the final task that we care about. 
+
+We support both Full-parameter checkpoints, and LoRA-adapter checkpoints. However, we recommend not combining the two by training a full-parameter model followed by a LoRA adaptation, because serving the resulting LoRA adapter will require the full-parameter checkpoint. Unless you are fine-tuning many such LoRA adaptors for different tasks this serving architecture does not have the neither the economical benefits of LoRA nor the quality benefits of full-parameter.
+
+## How to fine-tune from a previous checkpointing
+
+
+```python
+# [Optional] You can set the WandB API key to track model performance
+# !export WANDB_API_KEY={YOUR_WANDB_API_KEY}
+
+# Continue LoRA fine-tuning on the GSM8k dataset with Llama 3 8B
+!llmforge dev finetune llama-3-8b.yaml
+```
+
+## What we are fine-tuning on
+
+Running the above command will fine-tune on the [GSM8k dataset](https://huggingface.co/datasets/gsm8k). 
+In this example, we split the dataset into two halfs, each consisting of approximately 4.000 samples.
+The provided initial checkpoint has been trained on the first half and is already good at solving GSM8k. By running the above command, you continue fine-tuning from the provided checkpoint with the second half.
+
+Note the following evaluation losses. The first graph shows the evaluation loss on three epochs of training on the first half of the GSM8k dataset.
+
+<img src="https://raw.githubusercontent.com/anyscale/templates/main/templates/fine-tune-llm_v2/cookbooks/continue_from_checkpoint/../../assets/3epochs_1st_dataset.png" alt="evaluation loss of 1st training" width="700"/>
+
```

Comment:
Is it possible to overlay the two images? and add legends? instead of two separate curves?